tests/physics/helicity_formalism/angular_distributions/epem_to_dm_d0_pip_decay/test_distributions.py

```diff
@@ -0,0 +1,160 @@
+# cspell:ignore dphi, epem
+# pylint: disable=import-outside-toplevel,redefined-outer-name
+
+import os
+from math import cos
+from typing import Any, Callable, List, Optional, Tuple
+
+import numpy as np
+import pytest
+
+
+# Use this function to reproduce the model file.
+# Note the normalization part has been removed!
+def generate_model() -> None:
+    from expertsystem.amplitude.helicity_decay import (
+        HelicityAmplitudeGenerator,
+    )
+    from expertsystem.io import load_pdg, write
+    from expertsystem.particle import Parity, Particle
+    from expertsystem.reaction import generate
+
+    epem = Particle(
+        name="EpEm",
+        pid=12345678,
+        mass=4.36,
+        spin=1.0,
+        parity=Parity(-1),
+        c_parity=Parity(-1),
+    )
+    particles = load_pdg()
+    particles.add(epem)
+
+    result = generate(
+        initial_state=[("EpEm", [-1])],
+        final_state=[("D0", [0]), ("D-", [0]), ("pi+", [0])],
+        allowed_intermediate_particles=["D(2)*(2460)+"],
+        allowed_interaction_types="em",
+        particles=particles,
+    )
+
+    generator = HelicityAmplitudeGenerator()
+    amplitude_model = generator.generate(result)
+    amplitude_model.dynamics.set_non_dynamic("D(2)*(2460)+")
+    write(amplitude_model, "model.yml")
```

Comment:
Better use
```python
write(amplitude_model, output_dir + "model.yml")
```
and import that fixture with
```python
def generate_model(output_dir: str) -> None:
```

Reply:
but: why is this model also committed?

Reply:
See also the comment above.

I want the test to be as precise as possible, hence I keep the input fixed. So that if this test fails, it cannot be due to some change in the created model file. Instead the amplitude building or the data generation can only be responsible.

Reply:
In the `expertsystem` we want to move away from reading/writing in tests, because that causes issues when running tests in parallel, see e.g. https://github.com/ComPWA/expertsystem/issues/385#issuecomment-734440965. In addition, I think what are testing here is rather the I/O of the `expertsystem` (internally, `tensorwaves` just uses the `AmplitudeModel` class and uses `expertsystem.io` to read/write).